# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/browsers/windows/auxiliary-browsing-contexts/opener-setter.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/html/browsers/windows/auxiliary-browsing-contexts/opener-setter.html.ini
--- a/testing/web-platform/meta/html/browsers/windows/auxiliary-browsing-contexts/opener-setter.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/html/browsers/windows/auxiliary-browsing-contexts/opener-setter.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -2,12 +2,14 @@
   disabled:
     if (os == "linux") and (bits == 32) and debug: https://bugzilla.mozilla.org/show_bug.cgi?id=1483696
   expected:
-    if not debug and (os == "linux") and not webrender and (processor == "x86_64"): ["OK", "TIMEOUT"]
-    if debug and (os == "win") and not webrender: ["OK", "TIMEOUT"]
-    if debug and (os == "linux"): ["OK", "TIMEOUT"]
+    if (os == "linux") and not fission and webrender and debug: ["OK", "TIMEOUT"]
+    if (os == "linux") and fission and debug: ["OK", "TIMEOUT"]
+    if (os == "linux") and not fission and not webrender: ["OK", "TIMEOUT"]
+    if (os == "win") and debug and not webrender: ["OK", "TIMEOUT"]
   [Auxiliary browsing context created via `window.open` and setting `window.opener` to `null` should report `window.opener` `null`]
     expected:
-      if not debug and (os == "linux") and not webrender and (processor == "x86_64"): ["PASS", "TIMEOUT"]
-      if debug and (os == "win") and not webrender: ["PASS", "TIMEOUT"]
-      if debug and (os == "linux"): ["PASS", "TIMEOUT"]
+      if (os == "linux") and not fission and webrender and debug: ["PASS", "TIMEOUT"]
+      if (os == "linux") and not fission and not webrender: ["PASS", "TIMEOUT"]
+      if (os == "linux") and fission and debug: ["PASS", "TIMEOUT"]
+      if (os == "win") and debug and not webrender: ["PASS", "TIMEOUT"]
 